app/src/main/res/layout/content_event.xml

```diff
@@ -240,7 +240,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_margin="@dimen/layout_margin_large"
-                android:background="@color/colorPrimary"
+                android:background="@drawable/rounded_corner_button"
```

Comment:
That's the reason it's looking like this. Don't ever use background on buttons, use tint